README.md

```diff
@@ -1 +1,9 @@
-# ClassicPress-Migration-Plugin
\ No newline at end of file
+# Switch to ClassicPress from WordPress
+
+Switch to ClassicPress from WordPress and say Goodbye to Gutenberg. (ClassicPress Migration Plugin)
+
+https://www.ClassicPress.net/ is a new LTS fork actor of fomous CMS.
```

Comment:
Please follow the brand guidelines for the web address: https://www.classicpress.net/brand-guidelines/

Please change to "ClassicPress is a long term support (LTS) fork of WordPress 4.9.x with the goal of providing ongoing security patches, bug fixes and compatability updates for at least 2 years, giving you peace of mind that the WordPress you know and love can be used for the foreseeable future."

Reply:
OK to change text